frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -183,34 +187,37 @@ function EmbedFrame({
               </DashboardTabsContainer>
             )}
             <Separator />
-            {hasParameters && (
-              <ParametersWidgetContainer data-testid="dashboard-parameters-widget-container">
-                <ParametersFixedWidthContainer
-                  data-testid="fixed-width-filters"
-                  isFixedWidth={dashboard?.width === "fixed"}
-                >
-                  <SyncedParametersList
-                    question={question}
-                    dashboard={dashboard}
-                    parameters={getValuePopulatedParameters({
-                      parameters,
-                      values: _.isEmpty(draftParameterValues)
-                        ? parameterValues
-                        : draftParameterValues,
-                    })}
-                    setParameterValue={setParameterValue}
-                    hideParameters={hideParameters}
-                    setParameterValueToDefault={setParameterValueToDefault}
-                    enableParameterRequiredBehavior={
-                      enableParameterRequiredBehavior
-                    }
-                  />
-                  {dashboard && <FilterApplyButton />}
-                </ParametersFixedWidthContainer>
-              </ParametersWidgetContainer>
-            )}
           </Header>
         )}
+        {hasParameters && (
+          <ParametersWidgetContainer
+            hasVisibleParameters={hasVisibleParameters}
```

Comment:
since we already know we don't need to render the parameter should we not render the `ParametersWidgetContainer` instead of removing the padding via CSS?

We also do not render the parameter on the dashboard instead of using CSS magic.

https://github.com/metabase/metabase/blob/8ffcc27f275948a852e87d667144b29b1b2fe597/frontend/src/metabase/dashboard/components/Dashboard/Dashboard.tsx#L527-L543